src/libre/dialect/pcre/lexer.lx

```diff
@@ -275,12 +278,28 @@
 	/[0-9]+/ -> $count;
 }
 
+'#'  -> $maybe_comment;
+
+" "  -> $whitespace;
+"\t" -> $whitespace;
+"\v" -> $whitespace;
+"\f" -> $whitespace;
+"\n" -> $newline;
+"\r" -> $newline;
+
+# TODO: if there is no closing \E, the \Q continues
+# until the end of input.
```

Comment:
lx does actually have syntax for this, a zone which can never be exited from:
```
'\Q' {
    ...
}
```

@dhobsd behold! I added this for you, if you remember

Reply:
We talked about this on irc; it surprises me. but okay, if it works...